### PR TITLE
Update create_project.py

### PR DESCRIPTION
**Pull request by did0u007**
Fix Windows compatibility

### Description of the problem

First code was able to work perfectly on iOS and Linux but not on windows  because of paths and architectures are not the same  

### Describe the algorithm of actions that leads to the problem

Description of the algorithm of actions

### Screenshots of the problem

Sans titre

### Description of Changes

I did handle Windows paths and, multiples pythons versions and taking care with venvs.

### CODE REVIEW

**Comment by HeaTTheatR** [comment]
@did0u007 We strictly adhere to the `PEP8` standards. In your changes, this standard is completely ignored. After making the changes, run the `pre-commit run --all-files` command to automatically format the code according to the `PEP8` standards.

File: kivymd/tools/patterns/create_project.py [hunk 1]
```diff
@@ -657,13 +657,25 @@ def main():
...
+        envs = [i.strip("\\") for i in os.getenv("PATH").split(";") if i != ""]
+        for i in envs:
+            env = [j.lower() for j in i.split("\\")]
+            if python_version.replace(".", "").lower() == env[-1]:
+                if os.path.exists(os.path.join(i, "python.exe")):
+                    python_version = os.path.join(i, "python.exe")
+                else:
+                    parser.error("Python version you entred must be installed")
+                break
+
```
Comment (HeaTTheatR) [inline kivymd/tools/patterns/create_project.py:673]: I don't quite understand the meaning of your changes and how they relate to the error in the screenshot. Could you describe in more detail the problems that gets in OS Windows and how your changes solve these problems. Thanks.

Comment (did0u007) [reply kivymd/tools/patterns/create_project.py:673]: <sub>


    # Fixed to work on Windows
    if os.name == "nt":               # chek for Windows OS

        # get full path to project dir cuz this type of path "./path/to/project"  will cause problems if cmd get lunched from
        # different dir so path must be like this "c:/path/to/project"
        project_directory = os.path.realpath(args.directory)

        # get access to Variables Environment to check all pythons versions installed paths
        envs = [i.strip("\\") for i in os.getenv("PATH").split(";") if i != ""]


        for i in envs:
            env = [j.lower() for j in i.split("\\")]
            if python_version.replace(".", "").lower() == env[-1]: 
            
                 # in Windows Ex: python3.10 will be in "../Python310/python.exe " Dir and check if is really installed not only 
                 # an entry in OS Env list 
                if os.path.exists(os.path.join(i, "python.exe")):
                    python_version = os.path.join(i, "python.exe")
                else:
                    parser.error("Python version you entred must be installed")
                break
</sub>

Comment (HeaTTheatR) [reply kivymd/tools/patterns/create_project.py:673]:
```
    # get full path to project dir cuz this type of path "./path/to/project"  will cause problems if cmd get lunched from
    # different dir so path must be like this "c:/path/to/project"
```

The path to the project is specified by the user. And all responsibility for the correctness of the path to the project lies with the user.

**Comment by HeaTTheatR** [comment]
The PR is deprecated due to the new API of the KivyMD library 2.0.0 version

**Pull request closed without merging**